stl/inc/cmath

```diff
@@ -1318,7 +1319,14 @@ _NODISCARD constexpr _Ty _Common_lerp(const _Ty _ArgA, const _Ty _ArgB, const _T
         }
 
         // exact at _ArgT == 0, monotonic except near _ArgT == 1, bounded, determinate, and consistent:
-        const auto _Candidate = _ArgA + _ArgT * (_ArgB - _ArgA);
+        const auto _Candidate = [=]() constexpr {
+            if (_STD is_constant_evaluated()) {
+                return _ArgA + _ArgT * (_ArgB - _ArgA);
+            }
+            return _STD fma(_ArgT, _ArgB - _ArgA, _ArgA);
+        }
+        ();

```

Comment:
clang-format doesn't understand immediately invoked lambda :(

Reply:
I would prefer to see this given a named function because naming it tends to reduce duplication between translation units.